### PR TITLE
Update mail invitations: inner loop catches exception and put count a…

### PR DESCRIPTION
**Pull request by reiterl**
…nd index into the message.

### CODE REVIEW

File: tests/system/action/user/test_send_invitation_email.py [hunk 1]
```diff
@@ -208,7 +208,7 @@ def test_SMTPSenderRefused_not_authenticated(self) -> None:
             )
         self.assert_status_code(response, 200)
         self.assertIn(
-            f"SMTPSenderRefused: (530, b'5.7.0 Authentication required', '{EmailSettings.default_from_email}')",
+            f"Exception: (530, b'5.7.0 Authentication required', '{EmailSettings.default_from_email}') Count 1, Index 0",
```
Comment (r-peschke) [inline tests/system/action/user/test_send_invitation_email.py:211]: This should not occur. Please catch the SMTPSenderRefused in the inner part and throw it again to be caught in outer asserts. Reason: This occurs in sending email in **connected state**, but the sender is always the same and there is no need to assert each of the mails separately.
You'll stay with the problem of writing a test for this. You could use the AIOHandler in tests/system/action/mail_base.py and raise an Exception for a special address in mail. But that's on your fantasy;-)

File: openslides_backend/action/actions/user/send_invitation_email.py [hunk 2]
```diff
@@ -87,11 +87,18 @@ def perform(
...
+                        result["message"] = f"Exception: {str(e)}"
 
                     if result["sent"]:
                         instance.pop("meeting_id", None)
                         write_request = self.create_write_requests(instance)
                         self.write_requests.extend(write_request)
+                    else:
+                        result["message"] = (
+                            str(result["message"])
+                            + f" Count {len(list(action_data))}, Index {self.index}"
```
Comment (r-peschke) [inline openslides_backend/action/actions/user/send_invitation_email.py:100]: Proposal: f"Mail {self.index+1} from {len(action_data)}" for the index-part of the message.

**Review by r-peschke — CHANGES_REQUESTED** [review]

File: openslides_backend/action/actions/user/send_invitation_email.py (hunk 2 above)
Comment (reiterl) [reply openslides_backend/action/actions/user/send_invitation_email.py]: Done.

File: tests/system/action/user/test_send_invitation_email.py (hunk 1 above)
Comment (reiterl) [reply tests/system/action/user/test_send_invitation_email.py]: I have reraised the SMTPSenderRefused Exception. And cleanup the code.

**Review by r-peschke — APPROVED** [review]

**Pull request merged**